Makefile

```diff
@@ -9,6 +9,10 @@ MANINSTALLDIR=${PREFIX}/share/man
 #BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
 GO_MD2MAN ?= /usr/bin/go-md2man
 
+ifeq ($(DEBUG), 1)
+  override GOGCFLAGS += -N -l
```

Comment:
Why these changes? 
Without the quotes around -N -l, make all DEBUG=1 fails.
Without the else case to set GOGCFLAGS to "" make all fails.